docs/src/examples/RateSystem.md

```diff
@@ -0,0 +1,97 @@
+# Studying R-Tipping
+
+Let us explore a simple prototypical example of how to use the R-tipping functionality of this package.
```

Comment:
R-tipping isn't defined anywhere until here. Probably best to call this "rate dependent tipping".

On a side-note: there is no educational resources (to my knowledge) that properly teaches R-tipping to a non-PhD. The best attempt is a paper from Ritchie et al 2023, but it is still advanced (research article). I am currently working on one which will come in v2 of my textbook. I will try to upload some notes on GitHub in the meantime to cite here.